chromium_src/components/qr_code_generator/dino_image.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_QR_CODE_GENERATOR_DINO_IMAGE_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_QR_CODE_GENERATOR_DINO_IMAGE_H_
+
+// Contains constants clients use to to render a lion on top of a QR image.
+namespace dino_image {
+
+// Width of the lion pixel data.
+static constexpr int kDinoWidth = 24;
+// Height of the lion pixel data.
+static constexpr int kDinoHeight = 24;
+// Height of the lion pixel data, top segment.
+static constexpr int kDinoHeadHeight = 1;
+// Height of the lion image data, body segment.
+static constexpr int kDinoBodyHeight = kDinoHeight - kDinoHeadHeight;
+// Width of the lion image data.
+static constexpr int kDinoWidthBytes = (kDinoWidth + 7) / 8;
+
+// Pixel data for the lion's head (dino facing right - inapplicable to the lion)
+static const unsigned char kDinoHeadRight[kDinoWidthBytes * kDinoHeadHeight] = {
+    // clang-format off
+  0b00000000, 0b11111111, 0b00000000,
+    // clang-format on
+};
+
+// Pixel data for the lion's head (dino facing left - inapplicable to the lion)
+static const unsigned char kDinoHeadLeft[kDinoWidthBytes * kDinoHeadHeight] = {
```

Comment:
Aesthetically, I think the lion looks a little off. Would we be able to reproduce the image provided in [the original issue](https://github.com/brave/brave-browser/issues/11606)? Something like the following? Or are there dimensional constraints that make this difficult?

![image](https://user-images.githubusercontent.com/815158/119760688-0dad2e00-be70-11eb-9eff-ddbba08f0cc5.png)

Reply:
Here's a side-by-side comparison: https://jsfiddle.net/jonathansampson/r732dLux/23/

![image](https://user-images.githubusercontent.com/815158/119763562-67fcbd80-be75-11eb-959f-9e2079b26b8e.png)

Reply:
Yeah, there is a [comment](https://source.chromium.org/chromium/chromium/src/+/main:chrome/services/qrcode_generator/qrcode_generator_service_impl.cc;l=84) in the code that draws the image :
<pre>
  // If we request too big a dino, we'll clip. In practice the dino size
  // should be significantly smaller than the canvas to leave room for the
  // data payload and locators, so alert if we take over 25% of the area.
</pre>

In the issue the size is 55x55. The dino is 20w x 22h. The one I am using is 24x24, which I actually need to change to 24x25 since there is an [expectation in the code](https://source.chromium.org/chromium/chromium/src/+/main:chrome/services/qrcode_generator/qrcode_generator_service_impl.cc;l=37), which I didn't notice before, that the image height is > than the image width.

Reply:
24x26
```
0b00000000, 0b11111111, 0b00000000
0b00000001, 0b11111111, 0b10000000
0b00000011, 0b11111111, 0b11000000
0b00011111, 0b11111111, 0b11111000
0b00111100, 0b11000011, 0b00111100
0b00111100, 0b00000000, 0b00111100
0b00111000, 0b00000000, 0b00011100
0b00110001, 0b11100111, 0b10001100
0b01110000, 0b00100100, 0b00001110
0b00110000, 0b00100100, 0b00001100
0b00111000, 0b00000000, 0b00011100
0b00111100, 0b01000010, 0b00111100
0b00111110, 0b00111100, 0b01111100
0b00011110, 0b00011000, 0b01111000
0b00011100, 0b00000000, 0b00111000
0b00011110, 0b00011000, 0b01111000
0b00011111, 0b01100110, 0b11111000
0b00001111, 0b10000001, 0b11110000
0b00001111, 0b10000001, 0b11110000
0b00001111, 0b11000011, 0b11110000
0b00000111, 0b11111111, 0b11100000
0b00000111, 0b11111111, 0b11100000
0b00000001, 0b11111111, 0b10000000
0b00000000, 0b11111111, 0b00000000
0b00000000, 0b00111100, 0b00000000
0b00000000, 0b00011000, 0b00000000

```

Reply:
Thanks @jonathansampson. Will happily use yours 😄

Reply:
Apologies for the confusion—the original graphic was ~2x too large, but only because it was comprised of 2x2 cells 🙂

Reply:
Great job on this, folks! 😍

Reply:
Pushed an update. Ended up having to shave off 1 row off the top and the bottom due to the drawing peculiarities. The final version looks like this:

![image](https://user-images.githubusercontent.com/41635752/119845897-a7f48c80-bed7-11eb-85a1-d5435fd0f9b8.png)

Reply:
@mkarolin This isn't going to work. Can you give me the pixel dims and I will post a replacement?

Reply:
@bradleyrichter, sure: 22w x 24h.

Reply:
The final image:

![image](https://user-images.githubusercontent.com/41635752/120042373-f1c49c00-bfd7-11eb-8c4b-5da3575c0315.png)